src/frontend/packages/core/sass/_all-theme.scss

```diff
@@ -129,6 +130,8 @@ $side-nav-light-active: #484848;
   @include metrics-range-selector-theme($theme, $app-theme);
   @include app-multiline-title-theme($theme, $app-theme);
   @include tile-selector-theme($theme, $app-theme);
+  @include page-side-nav-theme($theme, $app-theme);
+  // @include cf-admin-add-user-warning($theme, $app-theme);
```

Comment:
Although not part of this PR this line should be included (I'd mistakenly left it out)

Reply:
You mean it should not be commented out?

Reply:
Correct